metadata-ingestion/docs/sources/powerbi/powerbi_pre.md

```diff
@@ -101,3 +103,47 @@ combine_result
 By default, extracting endorsement information to tags is disabled. The feature may be useful if organization uses [endorsements](https://learn.microsoft.com/en-us/power-bi/collaborate-share/service-endorse-content) to identify content quality.
 
 Please note that the default implementation overwrites tags for the ingested entities, if you need to preserve existing tags, consider using a [transformer](../../../../metadata-ingestion/docs/transformer/dataset_transformer.md#simple-add-dataset-globaltags) with `semantics: PATCH` tags instead of `OVERWRITE`.
+
+## Access Vs Data Ingestion
```

Comment:
Admin vs Data Ingestion ??

Reply:
I think `Admin Access vs Data Ingestion` would be good